src/command.c

```diff
@@ -941,6 +941,10 @@ void cmd_stats(struct command *cmd, int64_t end_time)
         latency = cmd->rep_time[1] - cmd->rep_time[0];
     }
     ATOMIC_INC(ctx->stats.remote_latency, latency);
+
+    if (config.slow_threshold >= 0 && latency > config.slow_threshold * 1000) {
```

Comment:
```
if (!STAILQ_EMPTY(&cmd->sub_cmds)) {
    first = STAILQ_FIRST(&cmd->sub_cmds);
    last = STAILQ_LAST(&cmd->sub_cmds, command, sub_cmd_next);
    latency = last->rep_time[1] - first->rep_time[0];
} else {
    latency = cmd->rep_time[1] - cmd->rep_time[0];
}
```
Do you mean this?
So should we use the time `latency = end_time - cmd->parse_time;` instead?